Makefile.toml

```diff
@@ -108,16 +108,17 @@ extend = "api-tests-flow"
 
 [tasks.format]
 workspace = false
+args = ["+nightly-2019-04-30", "fmt"]
 
 ################
 # Custom tasks #
 
 [tasks.check-rs-format]
 description = "Runs cargo fmt to check appropriate code format."
 workspace = false
-install_crate = { crate_name = "rustfmt-nightly", rustup_component_name = "rustfmt-preview", binary = "rustfmt", test_arg = "--help" }
-command = "cargo"
-args = ["fmt", "--", "--check"]
+script = [
```

Comment:
Are you sure this work? Your build is broken, probably related to that.

Reply:
Yes I tested it locally.
I also opened #1046 because cargo-make is so annoying :sweat_smile: 

The build failed because in this branch, `noise_handshake` is still included. That is why I added the line in the PR description that it depends on #1045. I rebased now so the build should be green after that.